lib/git_repository/history.rb

```diff
@@ -70,6 +72,17 @@ def commits(start_oid: nil, stop_oid: nil, path: nil, limit: nil, offset: 0, wal
     end
   end
 
+  def rugged_walk_order(walk_order)
```

Comment:
I personally prefer to use constant-hashes instead of case-statements if possible. What do you think?